### PR TITLE
Add eslint rules and fix small mistakes

### PR DESCRIPTION
**Pull request by Neicigam**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -9,6 +9,17 @@
         "before": "readonly",
         "beforeEach": "readonly"
       }
+    },
+    {
+      "files": ["**/*.js"],
```
Comment (KittySparkles) [inline .eslintrc:14]: Isn’t this the default?

File: .eslintrc [hunk 2]
```diff
@@ -9,6 +9,17 @@
...
+      "excludedFiles": ["bin/*"],
```
Comment (KittySparkles) [inline .eslintrc:15]: I think you should be able to do:
```suggestion
      "excludedFiles": ["bin"],
```

File: .eslintrc [hunk 3]
```diff
@@ -9,6 +9,17 @@
...
+      "rules": {
+        "no-console": "warn",
```
Comment (KittySparkles) [inline .eslintrc:17]: Let’s be strict.
```suggestion
        "no-console": "error",
```

Comment (Neicigam) [reply .eslintrc:17]: 😅

File: .eslintrc (hunk 1 above)
Comment (Neicigam) [reply .eslintrc:14]: I'm getting Missing property "files".

Comment (Neicigam) [reply .eslintrc:14]: Now I could probably remove this warning too 😄

File: .eslintrc (hunk 2 above)
Comment (Neicigam) [reply .eslintrc:15]: I've tried doing this initially but it doesn't work

**Pull request merged**